vercel.json

```diff
@@ -55,6 +55,14 @@
     {
       "source": "/docs/hardware/supported-hardware",
       "destination": "/docs/hardware/devices"
+    },
```

Comment:
TBH I'm not 100% sure how it's supposed to work.  I don't think I'm doing it the most efficient way.  The way I did it, I've found I needed the link with and without the trailing slash forwarded. @andrekir added some forwards using (.*) which I probably should have used.  Perhaps he knows.

https://vercel.com/docs/projects/project-configuration#redirects

I just tested a bunch of "dummy" urls to make sure they forwarded when I set up forwards originally.

Sorry I can't be more helpful on this one.

Reply:
:path* probably isn't necessary here because there aren't any subdirectories under licensing or trademark, though I'm also not sure if :path* would help with direct inks using # tags.

Reply:
The information is available to find either way.  If we find broken links in the future, it's easy to add more redirects.  I can't imagine anyone linking to these pages directly (other than search engines).

Reply:
had to add (.*) in the "source" so the old URLs ending with .html (or any extension) would work. otherwise:

https://meshtastic.org/docs/software/android/android-installation worked
https://meshtastic.org/docs/software/android/android-installation.html didn't

not familair with :path* all I know is it matches subfolders like "bluetooth" in https://docs/settings/config/bluetooth

anything after a # (called a fragment) is only local and not sent to the server (thats why we use it in the QR code).

Reply:
Ah that makes sense, of course .* is for extensions. Thanks @andrekir ! 
@rcarteraz you may want to add forwards with and without the trailing slash, though there must be a way to include both with one redirect.